.github/workflows/npm-audit.yml

```diff
@@ -0,0 +1,30 @@
+name: NPM Audit Scan for Vulnerabilities
+run-name: Running an Audit Scan with NPM
+on:
+  push:
+
+jobs:
+  npm-audit-scan-job:
+    name: NPM Audit Scan for Vulnerabilities
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
```

Comment:
We should remain consistent with our use of `yarn` as the package manager. You can add the line `cache: "yarn"`  to the `with:` section here to achieve that.